src/theme.ts

```diff
@@ -204,6 +205,22 @@ const themeOptions: ThemeOptions = {
         },
       },
     },
+    MuiListItemButton: {
+      styleOverrides: {
+        root: {
+          padding: '0 8px',
+          margin: 0,
+        },
+      },
+    },
+    MuiListItemText: {
+      styleOverrides: {
+        root: {
+          padding: '0 8px',
```

Comment:
This is affecting _all_ list items on the page, including `<ul>`s in blog post content. Probably meant to use a more local change.

Reply:
fixed